shell/src/main/java/org/apache/accumulo/shell/commands/FateCommand.java

```diff
@@ -243,7 +248,7 @@ private boolean cancelSubmittedTxs(final Shell shellState, String[] args)
       throws AccumuloException, AccumuloSecurityException {
     ClientContext context = shellState.getContext();
     for (int i = 1; i < args.length; i++) {
-      Long txid = Long.parseLong(args[i]);
+      Long txid = Long.parseLong(args[i], 16);
```

Comment:
Not sure why this is being auto-boxed here. `ManagerClient.cancelFateOperation(context, txid)` should also take a `long` not a `Long`, because by the time it gets to `FateService.Client.cancelFateOperation(...)` it is a `long`, not a `Long`. Using `Long` just unnecessarily auto-boxes and auto-unboxes.

```suggestion
      long txid = Long.parseLong(args[i], 16);
```

Reply:
> Also, was this a bug before that the radix wasn't 16?

Yes, Mike found it in testing.

Reply:
Addressed in c27679a